lib/npm/test.js

```diff
@@ -28,6 +28,9 @@ function authorName(author) {
 }
 
 function test(context, next) {
+  if (context.options.script) {
+    context.module.script = context.options.script;
```

Comment:
This is for the `citgm` (as opposed to `citgm-all`) case right? Could you add a comment in here noting that?